features/frameworks/utest/source/utest_shim.cpp

```diff
@@ -123,7 +123,6 @@ static int32_t utest_us_ticker_run()
             callback();
         }
     }
-    return 0;
```

Comment:
Will this not then complain about a function expecting to return a value but not supplying one?